src/main/java/hw3/components/AbstractComponent.java

```diff
@@ -0,0 +1,17 @@
+package hw3.components;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class AbstractComponent {
```

Comment:
In the name of the class you have mentioned that it `Abstract`. However you missed `abstract` modifier for class

Reply:
fixed. forgot to write